lnrpc/walletrpc/walletkit_server.go

```diff
@@ -88,6 +90,10 @@ var (
 			Entity: "onchain",
 			Action: "write",
 		}},
+		"/walletrpc.WalletKit/DeriveSharedKey": {{
+			Entity: "address",
```

Comment:
Or is this more `signer` permission? Maybe that is also the sub-server it needs to be in?

Reply:
Yes, definitely. Now that we have the secret key ring in the signer it makes much more sense to have this functionality there.